src/main/scala/lenthall/ErrorOr.scala

```diff
@@ -0,0 +1,22 @@
+package lenthall
+
+import cats.data.Validated.{Invalid, Valid}
+import cats.data.{NonEmptyList, Validated}
+
+object ErrorOr {
+  type ErrorOr[A] = Validated[NonEmptyList[String], A]
+
+  implicit class FlatMap[A](val fa: ErrorOr[A]) extends AnyVal {
+    /**
+      * Not consistent with `Applicative#ap` but useful in for comprehensions.
+      *
+      * @see http://typelevel.org/cats/tut/validated.html#of-flatmaps-and-xors
+      */
+    def flatMap[B](f: A => ErrorOr[B]): ErrorOr[B] = {
```

Comment:
We chatted about this in person a bit and punted to after standup tomorrow. What I said is that if there's something in core which shouldn't be accessible to the general system that to me this is an argument that that stuff shouldn't be in core, not to move stuff to another repo (a repo which IMO should be executed and then posthumously executed). I still view `core` as the place to stick stuff which might be useful throughout cromwell.